### ISSUE TITLE
OBS 26.1.0 saves the customized position of the docks but not their sizes

### ISSUE THREAD

**Issue by msmafra** [obsproject/obs-studio#4010]
OBS 26.1.0 (or 26.0.1 and even 25.0.8) do not save the sizes of the customized layout. It keeps the positions, but not the sizes. Reopening OBS the sizes (height or width) are always reset.

## Platform
Operating system and version: Fedora 33 (Workstation Edition)
OBS Studio version: OBS Studio - 26.1.0 (linux) installed with dnf (rpm package)

## Expected Behavior
To keep the width and heights of the docks. Keeping my layout the same way I left last usage.

## Current Behavior
Every time I open OBS I have to adjust height and width of docks even they being locked.
## Steps to Reproduce
1. Customize the width and/or heights of the docks (lock them or not);
2. Close OBS
3. Open OBS
4. Check that the changes are not there.
![Captura de tela de 2021-01-03 12-21-13](https://user-images.githubusercontent.com/1005457/103483755-3f8dfd00-4dc8-11eb-8ae0-ed52490d4271.png)
![Captura de tela de 2021-01-03 12-21-34](https://user-images.githubusercontent.com/1005457/103483757-40bf2a00-4dc8-11eb-9c05-31bac09c262d.png)

**Comment by msmafra** [comment]
I removed my plugins to check the other bug I was having (turned out was input overlay) and now not even the position of the scenes were saved, and they are all disabled. I tested via terminal, and if you open OBS and you use Ctrl + C (something about 3-4 times) during initialization, it loses which docks are enabled and their positions. As I was testing which plugin was making OBS hang during initialization, I did it a lot. I also repeated this after with the same results. And even if you choose another scene collection it resets all scene collections' docks positions.
